testmodels/src/main/java/com/amplifyframework/testmodels/commentsblog/BlogOwner.java

```diff
@@ -24,6 +25,9 @@
   private final @ModelField(targetType="ID", isRequired = true) String id;
   private final @ModelField(targetType="Blog") @HasOne(associatedWith = "owner", type = Blog.class) Blog blog = null;
   private final @ModelField(targetType="String") String wea;
+  private @ModelField(targetType = "AWSDateTime", isReadOnly = true) Temporal.DateTime createdAt;
```

Comment:
Should these two field be final?

Reply:
I don't think they can be, because they are not initialized in the constructor.  The only time they will actually be set is through Gson deserialization from the API response.